src/js/media-container.js

```diff
@@ -170,7 +170,8 @@ template.innerHTML = /*html*/`
       display: none;
     }
 
-    ::slotted([role=menu]) {
+    ::slotted([role="menu"]),
+    ::slotted([role="dialog"]) {
```

Comment:
I'd expect the default of a "dialog" to be centered on the player or aligned top, not the bottom.

Reply:
updated to be centered for role=dialog.
the vertical centering can't be done in the default slot.
that would have to be done via CSS or putting the dialog in the centered-chrome slot